JAICore/jaicore-ml/src/test/java/ai/libs/jaicore/ml/tsc/classifier/BOSSAlgorithmTest.java

```diff
@@ -16,28 +18,25 @@
 import ai.libs.jaicore.ml.tsc.dataset.TimeSeriesDataset;
 
 /**
- * @author Helen
- * DFT JUnit test
+ * @author Helen DFT JUnit test
  *
  */
 @RunWith(JUnit4.class)

```

Comment:
Is this really necessary? This might break if we upgrade JUnit at some point.

Reply:
removed this assertion